packages/cashier/src/stores/general-store.js

```diff
@@ -300,9 +303,14 @@ export default class GeneralStore extends BaseStore {
 
     async getAdvertizerError() {
         const advertiser_info = await this.WS.authorized.p2pAdvertiserInfo();
+        this.setAdvertiserInfo(advertiser_info.p2p_advertiser_info);
         this.setP2pAdvertiserError(getPropertyValue(advertiser_info, ['error', 'code']));
     }
 
+    setAdvertiserInfo(advertiser_info) {
```

Comment:
Why do we need this here in cashier? I have a PR to remove all P2P calls in cashier and adding this will affect that.

Reply:
Okay let me check. This is for showing the daily limit notification.

Reply:
@carol-deriv @farzin-deriv we can leave this for now since there's no other way to show the notification but by calling `p2p_advertiser_info` from cashier

Reply:
@farrah-deriv Can we fetch the `p2p_advertiser_info` somewhere in core instead? In the notification store maybe, As a temporary solution.

Currently, we are trying to remove all the usage of the cashier store outside of the cashier itself in [this](https://redmine.deriv.cloud/issues/84619#Farzin_/_R&D_/_Task_-_Remove_%60CashierStore%60_initialization_from_%60core%60_package) card and by the end, we will also remove the cashier store instantiation in the core package, So by adding the current implementation in this PR we will just create more work for refactoring.

If really there isn’t a way to move it to somewhere else then we can leave it as it is 🙇🏻